.github/workflows/chef-linear-notion-sync.yml

```
@@ -14,10 +16,7 @@ concurrency:
 
 jobs:
   sync:
-    uses: GroepOnline/OrgBeheer/.github/workflows/chef-dev-sync-reusable.yml@main
-    secrets:
-      LINEAR_API_KEY: ${{ secrets.LINEAR_API_KEY }}
-      NOTION_API_KEY: ${{ secrets.NOTION_API_KEY }}
-      NOTION_TOKEN: ${{ secrets.NOTION_TOKEN }}
-    with:
-      repo: ${{ github.event.repository.name }}
\ No newline at end of file
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)